habitat-baselines/habitat_baselines/il/data/nav_data.py

```diff
@@ -28,6 +37,8 @@
 if TYPE_CHECKING:
```

Comment:
In general, TYPE_CHECKING should be used sparingly and to avoid circular dependencies or occasionally heavy imports.